src/enclave/rpc_sessions.h

```diff
@@ -22,6 +22,9 @@ namespace enclave
   using ServerEndpointImpl = http::HTTPServerEndpoint;
   using ClientEndpointImpl = http::HTTPClientEndpoint;
 
+  static constexpr size_t max_open_sessions_soft_default = 1000;
+  static constexpr size_t max_open_sessions_hard_default = 1010;
```

Comment:
The defaults used to be set on the host if unspecified. This is now set here (enclave) if those have not been specified. 

Parallel questions: are these reasonable default values as they seem relatively low? See for example https://github.com/valyala/fasthttp#general-info-and-links which can apparently handle 1.5M connections,

Reply:
To be addressed by https://github.com/microsoft/CCF/issues/2612#issuecomment-973067527